packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/MultiFrameViewerContainer.js

```diff
@@ -137,7 +134,7 @@ function MultiFrameViewerContainer({
           src={img.src}
         >
           <SingleImageViewer
-            enableInteractionLayer={enableDrawing}
+            enableInteractionLayer={asyncStates.success && enableInteractionLayer}
```

Comment:
This check should be the same as original `loadingState === asyncStates.success && enableInteractionLayer`. Otherwise you're simply checking if `asyncStates.success` exists, which is always true.